server/src/main/java/io/crate/metadata/cluster/DropTableClusterStateTaskExecutor.java

```diff
@@ -50,6 +51,9 @@ protected ClusterState execute(ClusterState currentState, DropTableRequest reque
         RelationName relationName = request.tableIdent();
         Metadata currentMetadata = currentState.metadata();
 
+        if (!currentMetadata.contains(relationName)) {
```

Comment:
It's not clear to me, why this is now needed, could you please explain?

Reply:
The `getIndices` calls are a bit more lenient, so without it the action would succeed (without deleting anything as there is nothing) and the reported row count would be wrong:

```
[ERROR] Failures: 
[ERROR]   DDLIntegrationTest.testDropBlobTableIfExistsUnknownTable:775 
expected: 0L
 but was: 1L
[ERROR]   DDLIntegrationTest.testDropIfExistsUnknownTable:752 
expected: 0L
 but was: 1L
```